### PR TITLE
Project name and description

### LINKED ISSUE #1: Project needs to be renamed
Give the project a name
Give the project a description

### PR DESCRIPTION
**Pull request by connieelee**
Closes #1 

- describe the cool things you did in this branch! -

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "boilermaker",
+  "name": "cat-farm",
   "version": "1.0.0",
-  "description": "Some boilerplate code to get you started - get shakin'!",
+  "description": "Collin and Connie's lovely cat farm! Cats do grow on trees.",
```
Comment (collin) [inline package.json]: Wow! great description!

**Review by collin — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "boilermaker",
+  "name": "cat-farm",
```
Comment (collin) [inline package.json]: let's call it "super-cat-farm"

**Review by collin — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (connieelee) [reply package.json]: Thanks!

Comment (collin) [reply package.json]: Awesome. Great change!

File: package.json (hunk 2 above)
Comment (connieelee) [reply package.json]: @collin  how about now?

**Review by collin — APPROVED** [review]

**Pull request merged**